automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookClusterMonitoringSpec.scala

```diff
@@ -159,7 +159,7 @@ class NotebookClusterMonitoringSpec extends GPAllocFixtureSpec with ParallelTest
           .futureValue
 
         val request = defaultClusterRequest.copy(
-          jupyterDockerImage = None,
+          jupyterDockerImage = Some(LeonardoConfig.Leonardo.hailImageUrl),
```

Comment:
why are we using the hail image instead of base?

Reply:
This test is testing some hail method. We can probably just use base and check something else

Reply:
Yeah, I feel like this test should not use hail. Reading below, it looks like Hail verification is disabled anyway.

Reply:
yea, I wanted to check with you before I remove the hail part..is it okay to change this to base image, and what would be a good validation command? if nothing in particular, I'll just use something random

Reply:
Hm.. no strong opinion, something random is fine. This test is verifying that a cluster w/ preemptibles can be paused/resumed successfully, not any particular hail functionality.